src/thunderbots/software/ai/passing/pass.h

```diff
@@ -0,0 +1,101 @@
+/**
+ * Declaration for the "Pass" Class
+ */
+
+#include <array>
+#include <cstdlib>
+#include <iostream>
+
+#include "geom/point.h"
+#include "util/timestamp.h"
+
+namespace AI::Passing
+{
+    // TODO: simple test for getters
+    /**
+     * This class represents a Pass, with a given start position, end position,
+     * speed, and start time
+     */
+    class Pass
+    {
+       public:
+        /**
+         * Creates a pass with all 0 values
+         */
+        // Pass();
```

Comment:
Is this default constructor going to be used? (See comment about removing unused code above).

Reply:
nope, removed